src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -822,4 +822,7 @@ Additional information: {1}</value>
   <data name="Preference" xml:space="preserve">
     <value>Preference</value>
   </data>
+  <data name="The_Required_Style_of_the_first_rule_with_a_matching_Specification_will_be_enforced_at_the_chosen_Severity_level" xml:space="preserve">
+    <value>The Required Style of the first rule with a matching Specification will be enforced at the chosen Severity level.</value>
```

Comment:
I think "first" is still not clear.  I would explicitly mention that rules are tried from top to bottom.

Tagging @MattGertz and @DustinCampbell to try their hands at word-smithing too.